jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -380,6 +380,15 @@ Once you have configured the environmental variables and contexts, rerun the rea
 
 image::realitycheck-pipeline.png[Screenshot showing the realitycheck project building in the CircleCI app]
 
+==== Enabling Nomad Client for External Image Repository
+To enable the Nomad client so it can download images from an external or private repo, you can implement one of the two solutions suggested below:
```

Comment:
If this section gets moved to Configure and Deploy, it can probably refer to the `circleci/picard` image directly, to avoid confusion with all the other circleci images or the customers own images